_11ty/apply-csp.js

```diff
@@ -63,20 +65,35 @@ const addCspHash = async (rawContent, outputPath) => {
       hashes.push.apply(hashes, AUTO_RELOAD_SCRIPTS);
     }
 
-    const csp = dom.window.document.querySelector(
-      "meta[http-equiv='Content-Security-Policy']"
-    );
-    if (!csp) {
-      return content;
-    }
-    csp.setAttribute(
-      "content",
-      csp.getAttribute("content").replace("HASHES", hashes.join(" "))
-    );
-
     content = dom.serialize();
```

Comment:
We could consider to not do this. Writing the attribute to the DOM is for debugging purposes only.

Reply:
If I'm not mistaken, without writing the dom we don't have the csp-hash for inline scripts.

`<script csp-hash="">
    if (/Mac OS X/.test(navigator.userAgent))document.documentElement.classList.add('apple')
</script>`

Reply:
The hash here is inert. But it does help with debugging to compare the hash to the CSP content when things aren't working

Reply:
Ah ok, Now I understand the point. Thank you.